core/src/main/scala/org/bitcoins/core/protocol/tlv/TLV.scala

```diff
@@ -298,19 +319,79 @@ object RangeEventDescriptorV0TLV extends TLVFactory[RangeEventDescriptorV0TLV] {
   override def fromTLVValue(value: ByteVector): RangeEventDescriptorV0TLV = {
     val iter = ValueIterator(value)
 
+    val nonce = SchnorrNonce(iter.take(32))
     val start = Int32(iter.takeBits(32))
-    val stop = Int32(iter.takeBits(32))
+    val count = UInt32(iter.takeBits(32))
     val step = UInt16(iter.takeBits(16))
 
-    RangeEventDescriptorV0TLV(start, stop, step)
+    RangeEventDescriptorV0TLV(nonce, start, count, step)
+  }
+}
+
+case class LargeRangeEventDescriptorV0TLV(
+    base: UInt16,
+    isSigned: Boolean,
+    nonces: Vector[SchnorrNonce])
+    extends EventDescriptorTLV {
+
+  override val tpe: BigSizeUInt = LargeRangeEventDescriptorV0TLV.tpe
+
+  override val value: ByteVector = {
+    val isSignedByte = if (isSigned) ByteVector(0x01) else ByteVector(0x00)
```

Comment:
nit: Maybe factor out `0x01` and `0x00` into a constant somewhere with a nice name?